GNUmakefile

```diff
@@ -219,6 +219,12 @@ test: other-consul dev-build vet test-install-deps test-internal
 test-install-deps:
 	go test -tags '$(GOTAGS)' -i $(GOTEST_PKGS)
 
+go-mod-tidy:
+	@echo "--> Running go mod tidy"
+	@cd sdk && go mod tidy
+	@cd api && go mod tidy
```

Comment:
don't you have to go back to the parent directory?

Reply:
Nope! Each line of a makefile target evaluates in a brand new shell.